demisto_sdk/commands/common/errors.py

```diff
@@ -1143,6 +1144,15 @@ def release_notes_config_file_missing_release_notes(config_rn_path: str):
         return f'Release notes config file {config_rn_path} is missing corresponding release notes file.\n' \
                f'''Please add release notes file: {config_rn_path.replace('json', 'md')}'''
 
+    @staticmethod
+    @error_code_decorator
+    def release_notes_docker_image_not_match_yaml(rn_file_name, un_matching_files_list: list):
+        message_to_return = f'The {rn_file_name} release notes file contains incompatible Docker images:\n'
+        for un_matching_file in un_matching_files_list:
+            message_to_return += f"- {un_matching_file.get('name')}: Release notes file has dockerimage:"
+            message_to_return += f"{un_matching_file.get('rn_version')} but the YML file has dockerimage: {un_matching_file.get('yml_version')}\n"
+        return message_to_return
```

Comment:
Per Ido's suggestion, before we return the error let's also add a text directing users to run the format command on the RN file which is invalid.

Reply:
> Great! Per Ido's [suggestion](https://github.com/demisto/etc/issues/40160#issuecomment-1027930915) let's also add the ability to fix the RN docker version by running the format command. It will need to identify if there are unmatches in change the RN accoringly.

@ShahafBenYakir maybe we should do that via the RN update command?
Since the process we do is export content -> format -> generate docs + rn + etc.
(we need to format first because of _dev stuff which we don't want to see in docs or RN)

Reply:
Let's take this discussion offline